src/ITensorsExtensions/ITensorsExtensions.jl

```diff
@@ -1,5 +1,5 @@
 module ITensorsExtensions
-using LinearAlgebra: LinearAlgebra, eigen, pinv
+using LinearAlgebra: LinearAlgebra, eigen, ishermitian, pinv
```

Comment:
```suggestion
using LinearAlgebra: LinearAlgebra, eigen, pinv
```
From what I can tell this isn't needed since the function `ishermitian(...)` isn't being used anywhere.

Reply:
Good spot.